src/server/getters/getHackerForActiveHackathonFromSession.ts

```diff
@@ -17,21 +19,27 @@ const getHackerForActiveHackathonFromSession =
   async (): Promise<HackerFromSessionData> => {
     const session = await getServerSession(authOptions);
 
-    const hackathonId = await getActiveHackathonId(prisma);
-
+    let hackathonId = await getActiveHackathonId(prisma);
+    let closedPortal = false;
```

Comment:
You don't need this variable at all. Just return "true" or "false" directly where needed.

Reply:
I am setting it in the if Statement, to ensure that I return true only when it is needed.

Reply:
Yes but right after it you just return. So you can just replace this variable with direct boolean value in all of the return statements